lib/fastlane/plugin/revenuecat_internal/actions/pod_push_with_error_handling.rb

```diff
@@ -30,19 +30,23 @@ def self.run(params)
             output_str = e.message
 
             if output_str.include?("[!] Unable to accept duplicate entry for:")
-              FastlaneCore::UI.error("⚠️ Duplicate entry detected. Skipping push.")
-              return true
+              UI.error("⚠️ Duplicate entry detected. Skipping push.")
+              return false
```

Comment:
why this change? Seems to be making some tests fail as well

Reply:
I messed it up when fixing the merge conflict via GitHub. Fixed now :)